docker/autoscaler/requirements.txt

```diff
@@ -22,3 +22,4 @@ tabulate
 mlflow
 kubernetes
 boto3
+ipython==7.14.0
```

Comment:
If there's no specific reason for pinning this I'd prefer to just leave it at latest since I don't foresee it causing issues.

Reply:
Out of habit, I always pin versions, as it will inevitably cause some issue in my experience :) However, I just removed the version here.